style.css

```diff
@@ -1 +1,190 @@
-/*  Add your styling here */
+/* Add your styling here */
+body{
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+
+.logoup{
+    display: flex;
+    justify-content: center;
+    margin: auto;
+    width: 30%;
+}  
+
+/* cart icon */
+h3{
+    color: #ffffff;
+    float: right;
+    font-size: 16px;
+    padding: 0px 15px 0px 0px;
+
+}
+p{
+    display: flex;
+    justify-content: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    line-height: 1.5;
+    color:rgb(245, 245, 245);
+    background-color: rgb(17, 18, 19, 0.5);
+    padding: 0px 20px 0px 20px;
+}
+ul li{
+    float: left;
+    list-style: none;
+    position: relative;
+
+}
+a{
+   display: block; 
+   text-decoration: none;
+   color:white;
+   font-size: 22px;
+}
+ul li ul{
+    display: none;
+    position: absolute;
+    background-color:rgb(63, 8, 8,.8);
+    opacity: px;
+    padding: 20px;
+    border-radius: 0px 0px 5px 5px;
+}
+ul li:hover ul{
+    display: block;
+}
+ul li ul li{
+    width: 120px;
+}
+ul li ul li a{
+    padding: 15px 0px;
+}
+ul li ul li a:hover{
+   background-color: rgb(10, 10, 10, .7); 
+}
+body{
```

Comment:
be careful, there are now two instances of `body` being defined in this stylesheet - there should only be one. At the moment, the latest style definition for this html element will apply.